src/installers.ts

```diff
@@ -458,14 +462,20 @@ const giftwrapSourcesAgainIfNecessary = (
   api: VortexApi,
   treeInUse: ProcessedFileTree,
   instructions: VortexInstruction[],
-): VortexInstruction[] =>
-  (treeInUse.transform === Transform.Unwrapped
-    ? instructions.map(({ type, source, destination }: VortexInstruction) => ({
-      type,
-      source: path.join(treeInUse.wrapperDir, source),
-      destination,
-    }))
-    : instructions);
+): readonly VortexInstruction[] =>
+  (treeInUse.transform !== Transform.Unwrapped
+    ? instructions
+    : pipe(
+      instructions,
+      map((instruction) => (
+        instruction.source
+          ? {
+            ...instruction,
+            source: path.join(treeInUse.wrapperDir, instruction.source),
+          }
+          : instruction
+      )),
+    ));
```

Comment:
Could you walk me through this change? I don't quite see how it works, type wise I mean.

Reply:
does 
```js
{
  ...instruction,
  source: path.join(treeInUse.wrapperDir, instruction.source),
}
```
 basically unwrap `instruction` and then replace the `source` field of the object?

Reply:
Yes, it's like a chained `Object.assign`

Reply:
(And without having to worry about which other fields might exist.)